submissions/oluwatosinOke/html-forms/contactform.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <meta> CONTACT FORM <br>
+    </head>
+    <body>
+      <form action = "#" method =post entype="application/x-www-form-urlencoded"></form>
```

Comment:
Your `<form>` tag can not be empty. It should have some nested element within it